lib/Parser/Parse.cpp

```diff
@@ -8437,6 +8448,36 @@ ParseNodePtr Parser::ParseExpr(int oplMin,
     return pnode;
 }
 
+template<bool buildAST>
+void Parser::TrackAssignment(ParseNodePtr pnodeT, IdentToken* pToken)
+{
+    if (buildAST)
+    {
+        Assert(pnodeT != NULL);
```

Comment:
Done.

---
In reply to: [68135024](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68135024) [](ancestors = 68135024)